app/src/main/res/layout/contactpatient_callpatient.xml

```diff
@@ -91,7 +91,7 @@
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
     android:layout_marginHorizontal="@dimen/spacing_24"
-    android:layout_marginTop="@dimen/spacing_16"
+    android:layout_marginTop="@dimen/spacing_10"
```

Comment:
It looks like the margin between the call results divider and the call buttons section is not updated as well. Please take a look at it.

![Screenshot 2022-05-25 at 6.04.45 AM.png](https://graphite-user-uploaded-assets.s3.amazonaws.com/KqrtAphEKX3ow2MZW1lw/3d586d97-1b8c-4597-9952-71f784d297c6/Screenshot%202022-05-25%20at%206.04.45%20AM.png)